### ISSUE TITLE
Deep Neural Networks for Survival Analysis Using Pseudo Values

### ISSUE THREAD

**Issue by cmclean5** [cmclean5/PublicHealthModels#18]
## TL;DR

Paper proposes two step approach to model survival data using deep learning. In the first step, we transform each subject's survival time into a series of jackknife pseudo conditional survival probabilities and then use these pseudo probabilities as a quantitative response variable in the deep neural network model. By using the pseudo values, we reduce a complex survival analysis to a standard regression problem, which greatly simplifies the neural network construction. Our two-step approach is simple, yet very flexible in making risk predictions for survival data, which is very appealing from the practice point of view. 

### Paper Link

https://ieeexplore.ieee.org/document/9034100

### Author/Institution

Lili Zhao (University of Michigan), Dai Feng (Merck Research Laboratories).

## Overview

Paper proposes two step approach to model survival data using deep learning. In the first step, we transform each subject's survival time into a series of jackknife pseudo conditional survival probabilities and then use these pseudo probabilities as a quantitative response variable in the deep neural network model. By using the pseudo values, we reduce a complex survival analysis to a standard regression problem, which greatly simplifies the neural network construction. Our two-step approach is simple, yet very flexible in making risk predictions for survival data, which is very appealing from the practice point of view. 

## Methods

deep learning. survival analysis. competing risk events.

## Cite

L. Zhao and D. Feng, "Deep Neural Networks for Survival Analysis Using Pseudo Values," in IEEE Journal of Biomedical and Health Informatics, vol. 24, no. 11, pp. 3308-3314, Nov. 2020, doi: 10.1109/JBHI.2020.2980204.

## Comments

Source code is freely available at: http://github.com/lilizhaoUM/DNNSurv
